### DISCUSSION TITLE
How to search whether the specified string is contained in the array?

### DISCUSSION THREAD

**Discussion by kwaa** [lumeland/lume#405] [Q&A] [answered]
I use [my own Frontmatter format](https://fff.js.org), where [`flags: ['unlisted']`](https://fff.js.org/concepts/flags.html) means that this post is not included in the list.

So how do I accurately search for flags that contain unlisted? [The search plugin's documentation](https://lume.land/plugins/search/) doesn't seem to indicate this, or maybe it doesn't support it yet.

**Comment by oscarotero** [comment, marked as answer]
You can use `flags*=unlisted`. The `value*=otherValue` code is compiled to `value.includes(otherValue)`, that works with strings and arrays.

**Reply by kwaa** [reply]
tyvm! so can I use `flags!*=unlisted` to exclude posts?

**Reply by oscarotero** [reply]
Mmm, not. But that's a good idea. It would be nice to have this.

**Reply by kwaa** [reply]
> Mmm, not. But that's a good idea. It would be nice to have this.

Okay, I'll open a feature request.
